alphagov/paas-product-pages
# add new guidance on the use of GitHub pages for static content

- new section based on the paas tech docs that migrated to github pages
- updated the updates section

# who to review 
anyone but @pauldougan 

# how to review
- clone branch
- npm install && npm run local
- https://localhost:3000
- ensure content is readable (the target audience is a team with static docs based on TDT looking for help migrating their own content)
- ensure the links go somewhere sensible
